deploy/charts/rook-ceph-cluster/templates/deployment.yaml

```
@@ -96,6 +96,7 @@ spec:
               watch_endpoints
           imagePullPolicy: IfNotPresent
           tty: true
+          securityContext: {{- .Values.toolbox.containerSecurityContext | toYaml | nindent 12 }}
```

```suggestion
          securityContext: {{- .Values.toolbox.containerSecurityContext | toYaml | nindent 12 }}
```

toolbox and rook operator pod have totally different working, I think it's safe to have different values.

@travisn thoughts?

---

I do have different default values for the 2, mostly keeping to the current value to prevent any breaking changes.